tests/code-splitting/subtests/splitting-modules/subtests/multi-entry/webpack/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-result: fail
+result: pass
 ---
 
-Webpack extracts the common `objects.js` module as-is into its own shared bundle.
+Webpack optimizes modules separately for each runtime. In projects that do not require `runtimeChunk: "single"`, each entry is optimized to remove unreachable exports for that entry. It is important to note that configurations without `runtimeChunk: "single"` may [duplicate modules](https://bundlers.tooling.report/code-splitting/multi-entry/#webpack) used by multiple entries.
```

Comment:
@developit should this be a partial because it prevents workarounds to make multiple entry points on the same page work?

I mean, you'd get the same 'pass' in Rollup if you treated each entry point as a separate build.

Reply:
Hmm - yeah I could see marking this as a partial for that reason. Curious if @sokra would be okay with that - webpack would still be the only bundler that doesn't fail this test.

Reply:
Although it raises the question if Rollup should be partial here too, since it's pretty easy to do a build per entry point.

Reply:
@developit I think we agreed in the meeting to keep this as a fail (rather than make Rollup also a partial), but cover in the test that the solution shouldn't prevent module sharing between entry points, since that's kinda the point of the test.

Reply:
This is different from a separate build per entrypoint since separate builds do not share chunks, while a single build can do that.

The test case actually requires to be able to do that, as entry-1 and entry-2 both use the same exports of `objects.js` and the readme says it must be shared. (Technically here you could manually do a build where entry-2 and entry-3 are build together, but this fails when there is more sharing between the entries).

Note that there is more than `runtimeChunk: "single"` vs `runtimeChunk: false`. One could use `entry.xx.runtime` and `entry.xx.dependOn` on per entry basis to control which entrypoints are used on the same page (optimized together) and which are used in isolation (optimized independently). So it's not yes or no, but you can also have both.

Note that you can not have differently optimized variants of the same module on the same page, since this would duplicate state and instance equality.

Reply:
That's fair. Let's keep this as a pass. Thanks for the info!